evalit/nifi/nifi_automation.py

```diff
@@ -33,9 +35,13 @@ def __init__(
     ) -> None:
         super().__init__(config=config, files=files, debug=debug)
         self.nifi_url = nifi_url
+
+        assert os.path.exists(nifi_dir), f"{nifi_dir} path doesn't exist!"
```

Comment:
Just curious: How about a `try/except` instead?

Reply:
This is meant as a sanity check that nifi_dir should pre-exist. So I think try/catch doesn't make sense.